### PR TITLE
update pyladies content for community page

### PR DESCRIPTION
**Pull request by Lee-W**

### CODE REVIEW

File: src/templates/pycontw-2020/contents/zh/about/community.html [hunk 1]
```diff
@@ -24,9 +24,9 @@ <h2><a href="http://www.meetup.com/Tainan-py-Python-Tainan-User-Group/" target="
 <p>2013 年，公司從台北搬到台南的 Joe，順勢成立了 Tainan.py。適逢 jserv 移駕成大資工，Joe 為了壯大聲勢，決定與同在台南的 MOSUT 社群合辦聚會，成為「Tainan.py × MOSUT」。Tainan.py 的中場休息時間，會推廣台南在地美食，讓與會者品嘗鹽酥雞、杏仁湯等，讓衣服尺寸變大的食物。一個月聚會一次，時間在週末下午，是一種吃不夠還可以再續攤吃台南美食的概念。</p>
 
 
-<h2><a href="http://www.meetup.com/PyLadiesTW/" target="_blank" rel="noopener">PyLadies Taiwan</a></h2>
+<h2><a href="https://tw.pyladies.com/" target="_blank" rel="noopener">PyLadies Taiwan</a></h2>
 
-<p>2014 年，是 Python 台灣在地社群迅速擴張的一年。首先是伊瑪貓在 PyCon Taiwan 2013 夜市中成立的 PyLadies 社群，在台北舉行了第一次的活動。這是一個女孩限定的社群，不定期會舉辦活動，讓女生們彼此交流、交換 Python 的心得、聊天、認識朋友。據說只要是喜歡 Python 的女生、想學 Python 的女生、想瞭解 Python 的女生，都可以成為 PyLadies 的一員。</p>
+<p>PyLadies 是一個專屬於女生的 Python 愛好者聚會，專供喜歡 Python 的女生、想學 Python 的女生、想瞭解 Python 的女生，彼此交流、交換心得，聊天認識朋友的地方。我們是一個國際性的 Python 交流組織 ，主要任務是希望透過分享、教育與Workshop等活動的方式幫助更多女生可以在 Python 社群中成為主動的參與者與領導者。我們將會於每個月舉辦不同主題的活動，例如：初學者的入門、使用 Python 的應用與經驗分享，相關教學等主題。只要你是女孩並對 Python 有興趣，歡迎你來參加喔！</p>
```
Comment (uranusjr) [inline src/templates/pycontw-2020/contents/zh/about/community.html]: Should we update the English content to match this?

Comment (Lee-W) [reply src/templates/pycontw-2020/contents/zh/about/community.html]: I've sent an email to PyLadies for requesting an English version.

Comment (Lee-W) [reply src/templates/pycontw-2020/contents/zh/about/community.html]: I just update the English content.

**Comment by Lee-W** [comment]
@uranusjr I've already added the English script for it.

File: src/templates/pycontw-2020/contents/en/about/community.html [hunk 2]
```diff
@@ -25,7 +25,7 @@ <h2><a href="http://www.meetup.com/Tainan-py-Python-Tainan-User-Group/" target="
 
 <h2><a href="https://tw.pyladies.com/" target="_blank" rel="noopener">PyLadies Taiwan</a></h2>
 
-<p>2014 was a year that the number of Python local communities grew rapidly. One of these communities was PyLadies Taiwan, founded during the night market event in PyCon Taiwan 2013. The first PyLadies meetup was hosted in Taipei. It is a girls only community. Currently there are casual meetups, which provides a place to exchange experience in Python, chat, and meet new friends. Every girl who likes Python or wants to learn Python are welcomed to become a member of PyLadies Taiwan.</p>
+<p>Pyladies is a group of women developers who love the Python programming language. We are an international mentorship group  with a focus on helping more women become active participants and leaders in the Python open-source community. We host monthly meetups with different topics such as beginners meetups, project of python presentation and tutorial. Open to all who identify as women. Feel free to join us!</p>
```
Comment (uranusjr) [inline src/templates/pycontw-2020/contents/en/about/community.html]:
```suggestion
<p>PyLadies is a group of women developers who love the Python programming language. We are an international mentorship group  with a focus on helping more women become active participants and leaders in the Python open-source community. We host monthly meetups with different topics such as beginners meetups, project of python presentation and tutorial. Open to all who identify as women. Feel free to join us!</p>
```

**Comment by uranusjr** [comment]
One nitpick, otherwise LGTM.

**Comment by Lee-W** [comment]
It's been fixed.

**Pull request merged**